PubnativeLite/PubnativeLite/Interstitial/HyBidInterstitialPresenterFactory.h

```diff
@@ -27,6 +27,7 @@
 @interface HyBidInterstitialPresenterFactory : NSObject
 
 - (HyBidInterstitialPresenter *)createInterstitalPresenterWithAd:(HyBidAd *)ad
+                                                  withSkipOffset:(NSUInteger)skipOffset
```

Comment:
Do we need to add this withSkipOffset to the existing method or do we need to create a separate one so that we'll have 2 `createInterstitalPresenterWithAd` methods. One with `skipOffset` param, one without. What do you think @eros902002 ?🤔